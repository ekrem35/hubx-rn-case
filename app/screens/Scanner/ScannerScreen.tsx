import React from 'react';

import {View} from 'react-native';

export default function ScannerScreen() {
  return <View style={{flex: 1, backgroundColor: '#F1311150'}} />;
}
