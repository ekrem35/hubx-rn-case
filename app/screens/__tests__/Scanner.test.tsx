/**
 * @format
 */

import 'react-native';
import React from 'react';
import ScannerScreen from '../Scanner/ScannerScreen';

import {it} from '@jest/globals';

import {render, cleanup} from '@testing-library/react-native';

afterEach(cleanup);

describe('test scanner screen', () => {
  it('renders correctly', async () => {
    const screen = await render(<ScannerScreen />);

    expect(screen.toJSON()).toMatchSnapshot();
  });
});
