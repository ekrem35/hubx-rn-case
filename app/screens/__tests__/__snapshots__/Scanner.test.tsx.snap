// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test scanner screen renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#F1311150",
      "flex": 1,
    }
  }
/>
`;
